Fabric.Identity.API/scripts/tests/Install-IdPSS-Utilities.tests.ps1

```diff
@@ -395,5 +395,89 @@ Describe 'IdPSS Unit Tests' {
         }
     }
 }
+Describe "IdPSS Integration Tests"{
+    Describe 'Register-Identity' -Tag 'Integration'{
+        BeforeEach{
+            # Arrange 
+            # Add to the powershell TestDrive which cleans up after each context, leaving the tests folder configs unchanged
+            
+            $Global:testAzureFile = "testAzure.config"
```

Comment:
Only need one set of globals is needed outside the parent Describe block. I could also see consolidating to one BeforeEach and the child Describe block as 'Register functions'.  Two It blocks for Identity and IdPSS.